config/ufs/config_files.xml

```diff
@@ -122,8 +122,8 @@
   <entry id="COMP_ROOT_DIR_CPL">
     <type>char</type>
     <values>
+      <value component="cpl" >$SRCROOT/src/model/CMEPS</value>
```

Comment:
This is okay but i think still need to have clear distinction between mediator and driver under CIME. So, maybe we could introduce  COMP_ROOT_DIR_DRV etc.

Reply:
I have opened https://github.com/ESMCI/cime/issues/3712 to address this issue.

Reply:
Thanks. That is great